sample_project/app/v1/users/views.py

```diff
@@ -0,0 +1,119 @@
+from flask.views import MethodView
+from app.decorators import validate_request
+from app.common_utils import render_success_response
+from app.v1.users.service import SubApp1Service
+from flask import jsonify
+from flask import request
+from app.connections import SqlConnection
+import pymysql
+
+
+
+def db_connection():
+    # conn=mysql.connector.connect(host='localhost',user="root",password="root",database= "demo")
+    # cur=conn.cursor()
+
+
+    db_config = {
+        'host': 'localhost',
+        'user': 'root',
+        'password': 'root',
+        'database': 'db'
+    }
+
+
+    connection = pymysql.connect(**db_config)
+    return(connection) 
+db_connection()
+class GetUserName(MethodView):
+
+    @validate_request
+    def get(self, params, headers, *args, **kwargs):
+        response, message = SubApp1Service(params, headers).get_static_api_response()
+        return render_success_response(response, message)
+
+class GetUserDetails(MethodView):
+    def get(self):
+       conn = db_connection()
+       cursor = conn.cursor(pymysql.cursors.DictCursor)
+       query = "SELECT * FROM employee"
```

Comment:
NO DB calls in service file. All external call like db or external api will be via coordinator.